rotkehlchen/chain/evm/decoding/types.py

```diff
@@ -10,9 +10,17 @@ class CounterpartyDetails(NamedTuple):
     image: str
 
 
+class EventDirection(SerializableEnumNameMixin):
+    """Describes the mode/direction of asset's movement (to the user, from the user, or informational)"""  # noqa: E501
+    IN = auto()
+    OUT = auto()
+    NEUTRAL = auto()
+
+
 class EventCategoryDetails(NamedTuple):
     label: str
     icon: str
+    direction: EventDirection
     color: Optional[Literal['green', 'red']] = None
```

Comment:
This is not really for evm decoding but for all events now. Can you move it out to a better place? Maybe in the accounting folder?